### PR TITLE
Minor enhancements

### PR DESCRIPTION
**Pull request by royrapoport**

### CODE REVIEW

File: slack_user_report.py [hunk 1]
```diff
@@ -71,6 +73,18 @@ def make_report(self, ur, us, pur, pus, uid):
...
+        # blocks += self.firstpost(uid)
+        return blocks
+
+    def firstpost(self, uid):
+        blocks = []
+        entry = self.fp.get(uid)
+        if not entry:
+            return blocks
+        url = utils.make_url(entry['slack_cid'], entry['message_id'])
+        m = "By the way, your first-ever message (to the best of our knowledge) was {}"
```
Comment (jdq22) [inline slack_user_report.py:85]: Is the only case where we'd be wrong about their first message is when their first message is in a DM, MPDM, or private channel?

File: firstpost_processor.py [hunk 2]
```diff
@@ -26,12 +26,16 @@ def download(self):
...
-                                                              cid_count, cid, self.channel.get(cid)))
-            gotit = self.fp.get_channel(cid)
-            if gotit:
-                print("\tSkipping channel -- already got it")
+            channel_entry = self.channel.get(cid)
+            if channel_entry:
+                name = channel_entry['friendly_name']
+            else:
+                name = "Unknown"
+            if name != "announcements":
```
Comment (jdq22) [inline firstpost_processor.py]: Will you please help me understand why we'd move on if the channel is #announcements?

File: slack_user_report.py (hunk 1 above)
Comment (royrapoport) [reply slack_user_report.py:85]: *Probably* .  Hopefully :)

File: firstpost_processor.py (hunk 2 above)
Comment (royrapoport) [reply firstpost_processor.py]: That was for testing, and should be out of the final version of the code.

Comment (royrapoport) [reply firstpost_processor.py]: Oh, no, actually sorry it was still there -- submitted new change to take it out

**Comment by jdq22** [comment]
@royrapoport Is this ready for a final pass?

**Comment by royrapoport** [comment]
@jdq22 Yes it is. No more pending work

**Review by jdq22 — APPROVED** [review]

**Pull request merged**